modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,175 @@
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+
+* You have created the `cluster-monitoring-config ConfigMap` object.
+
+.Procedure
+
+. Enable monitoring in user-defined projects by adding the following configuration to the `cluster-monitoring-config` config map:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
+----
+
+. Configure the SMCP for user-workload monitoring:
++
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  addons:
+    prometheus:
+      enabled: false # <1>
+    grafana:
+      enabled: false # <2>
+  meshConfig:
+    extensionProviders:
+    - name: prometheus # <3>
+----
+<1> Disable the default Prometheus instance provided by OSSM.
+<2> Disable Grafana. It is not supported with an external Prometheus instance.
+<3> Configure the Prometheus extension provider.
+
+. Apply a custom network policy to allow ingress traffic from the `user-workload monitoring` namespace:
++
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: user-workload-access
+  namespace: bookinfo
+spec:
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          network.openshift.io/policy-group: monitoring
+  podSelector: {}
+  policyTypes:
+  - Ingress
+----
+
+. Apply a `Telemetry` object to enable traffic metrics in Istio proxies:
++
+[source,yaml]
+----
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: enable-prometheus-metrics
+  namespace: istio-system # <1>
+spec:
+  selector: # <2>
+    matchLabels:
+      app: bookinfo 
+  metrics:
+  - providers:
+    - name: prometheus
+----
+<1> A `Telemetry` object created in the control plane namespace applies to all workloads in a mesh. To apply telemetry to only one namespace, create the object in the target namespace.
+<2> Optional: Setting `spec.selector.matchLabels` applies the `Telemetry` object to specific workloads in the target namespace.
+
+. Apply a `ServiceMonitor` object to monitor the Istio control plane:
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: istiod-monitor
+  namespace: istio-system # <1>
+spec:
+  targetLabels:
+  - app
+  selector:
+    matchLabels:
+      istio: pilot
+  endpoints:
+  - port: http-monitoring
+    interval: 30s
+    relabelings:
+    - action: replace
+      replacement: "<smcp_name>-<smcp_namespace>" # <2>
+      targetLabel: <mesh_id>
+----
+<1> Since OpenShift Monitoring ignores the field `spec.namespaceSelector` in `ServiceMonitor` and `PodMonitor` objects, you need to apply the `PodMonitor` object in all mesh namespaces, including the control plane namespace.
```

Comment:
```suggestion
<1> Since {product-title} monitoring ignores the field `spec.namespaceSelector` in `ServiceMonitor` and `PodMonitor` objects, you need to apply the `PodMonitor` object in all mesh namespaces, including the control plane namespace.
```
make consistent with the earlier formatting here https://github.com/openshift/openshift-docs/pull/61854/files#diff-7e86933621f91bd09740edfa42e222d127081e41384db64c0beecd5d87ce685eR10